test/e2e-go/features/stateproofs/stateproofs_test.go

```diff
@@ -382,7 +382,7 @@ func TestStateProofMessageCommitmentVerification(t *testing.T) {
 }
 
 func getDefaultStateProofConsensusParams() config.ConsensusParams {
-	consensusParams := config.Consensus[protocol.ConsensusCurrentVersion]
+	consensusParams := config.Consensus[protocol.ConsensusFuture]
```

Comment:
We should remember to change this after the consensus upgrade. Maybe add a ticket?